### ISSUE TITLE
Filavandrel gets a child portrait

### ISSUE THREAD

**Issue by cirogionta** [JonStryker/Witcher#53]
He starts off with an elven portrait, but once you start the game his portrait changes to a childs. It's age related as I've adjusted the age on some other elven character's and it seems that once they get past 250 years old the portrait changes back to a child's one. So all the elves have some kind of Benjamin Button syndrome. 

Thanks :)

**Comment by JonStryker** [comment]
Portraits for old/immortal characters are bugged. I did a maintenance event that re-initializes portraits which worked okay. Maybe the event is broken now? Just check a month into the game with all these sorcerers. They should get the right portrait after a few days ingame.

**Comment by rquinio** [comment]
This one is a mystery to me, looks like the engine is going nuts, it's not even the portrait of elven child.
And there are immortal characters that are older (ex: Vezemir), that don't have the bug.

I've found the maintenance event for the sorcerers, and found a way to optimize it via on_startup.
Also found a bug in portraits making sorcerer use elfgfx ethnicity since 2.7

**Comment by rquinio** [comment]
Based on https://www.reddit.com/r/CrusaderKings/comments/6pi864/immortals_with_child_portraits/ :
- this happened starting with 2.7 for something like 5% of his immortal characters (wasn't the case before)
- giving a non-immortal 2000 years old character an immortal trait via event turns the portrait to a child
- that he resorted on not using immortal traits, and set NATURAL_DEATH_CHANCE_AGE_X to 0 instead & handle everything via events, with modifiers on health etc.

**Comment by rquinio** [comment]
Related bug report: https://forum.paradoxplaza.com/forum/index.php?threads/ck-ii-2-8-1-1-immortal-age-stack-overflow.1062061/

**Comment by TerryDuke** [comment]
I've also discovered this bug recently as well.  At least he's an _elf_ child now :)

This is likely due to the integer overflow problem.  Some possible solutions:

- Change his age so that he's less than 200 years old (which will give you about ~55 years before the image flips to a child again).
- Change his age so that he's a lot older (anything more than 270... which is only 5 extra years), for example the same age as Vezemir, may fix this.  The logic being that his age minus 256 results in a number that is considered an adult age.  Then again, if a 2000 year old is always displayed as a child, then perhaps this won't fix the issue...
- Change his image to a fixed portrait, since he is a canon figure.  This image is decent: http://custom-gwent.com/cards/7525de22ab16d730246b14f77014877b

![bug03-child_265_years_old](https://user-images.githubusercontent.com/39809872/40813154-548bc3b0-6507-11e8-9a33-24f14f141dea.png)

**Comment by TerryDuke** [comment]
I believe I have a fix for this now.  I'll create a Pull Request, once I better understand the contributor process and clean some stuff up.  I'm still learning modding, so I tried two things:
1. Changed the age.
2. Gave him a pre-generated portrait.

Solution 2 worked, so I don't need to test solution 1 anymore :-)

![filavandrel-fix](https://user-images.githubusercontent.com/39809872/40820402-532cc5f6-652e-11e8-979d-4686b7083a2b.png)
